features/android/android_callbacks.feature

```diff
@@ -38,3 +38,80 @@ Feature: Callbacks
         And the session payload field "device.osVersion" equals "Custom OsVersion"
         And the session payload field "device.runtimeVersions" is not null
         And the session payload field "device.totalMemory" equals 999
+
+     Scenario: On Send Native Callback
+        When I run the "Java Background Crash" mobile scenario
+        And I wait for 4 seconds
```

Comment:
Is there any way to avoid this wait? Adding an arbitrary sleep feels quite likely to lead to test flakes in future if the SDK isn't able to process the scenario in time

Reply:
@twometresteve any ideas here Steve?

Reply:
I spoke with steve and he said there are some examples in the unreal tests of detecting when the app is back online. I will make a seperate ticket to implement that in all unity tests that use sleep if possible 👍